llvm/test/Transforms/Util/loop-iter-count-asserts.ll

```diff
@@ -0,0 +1,295 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; NOTE: Example file for converting loop iter count to assumes in Loop
+;
+; This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+; See https://llvm.org/LICENSE.txt for license information.
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+;
+; (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+; unit test for the WAW register renaming pass and check edge cases
```

Comment:
nit: out-of-context comment.

Reply:
missed that template copying, adopted it